### PR TITLE
Add swagger GUI, docker compose v1 -> v3

### PR DESCRIPTION
**Pull request by ASVBPREAUBV**
Added an additional docker-compose.yml with swagger gui as own container. updated docker-compose.yml do version 3

### CODE REVIEW

**Comment by steve-chavez** [comment]
Thanks @ASVBPREAUBV , the new version and the swagger-ui are a good addition(just discovered I cannot have `network_mode: host` in v1) however the PR contains unwanted commits, can you `rebase` so I can review it better?

**Comment by ASVBPREAUBV** [comment]
Sorry for the late reply. Im new to open-source cooperation. can you maybe link to resources to help me with your request?

**Comment by steve-chavez** [comment]
Sure, basically you need to run these :
```bash
git clone https://github.com/ASVBPREAUBV/postgrest-docs
git remote add upstream https://github.com/PostgREST/postgrest-docs
git fetch upstream
git checkout patch-2 ## your pull request branch
git rebase upstream/master
git push -f 
```

**Comment by steve-chavez** [comment]
I think I'd be better if the swagger-ui container is added as an extra and optional step, so how about if we modify the change to be like this:
```
If you want to have a visual overview of your API you can add swagger-ui to your :code:`docker-compose.yml`:

.. code-block:: yaml

  swagger:
    image: swaggerapi/swagger-ui
    ports:
      - "8080:8080"
    expose:
      - "8080"
    environment:
      API_URL: http://localhost:3000/

With this you can see the swagger-ui on port 8080.
```

**Comment by steve-chavez** [comment]
Can you make the change? The rebase worked fine, now you need to go to the directory where you cloned the repo and then:
```bash
git checkout patch-2
## Go to the install.rst file with your favorite text editor(sublime, notepad, etc),
## make the change I proposed, and then
git commit -m 'Make swagger-gui an optional step in installation'
git push
```

**Comment by ASVBPREAUBV** [comment]
is this correct?

File: install.rst [hunk 1]
```diff
@@ -231,6 +231,28 @@ To avoid having to install the database at all, you can run both it and the serv
...
+    expose:
+      - "8080"
+    environment:
+      API_URL: http://localhost:3000/
+
+With this you can see the swagger-gui in your browser on port 8080.
+
+Go into the directory where you saved this file and run :code:`docker-compose up`.
+You will see the logs of both the database and PostgREST, and be able to access the latter on port 3000.
+Also you will be able to see the swagger overview on the port 8080.
```
Comment (steve-chavez) [inline install.rst]: Only thing missing is to remove this three lines that now are redundant.

File: install.rst [hunk 2]
```diff
@@ -229,7 +229,30 @@ To avoid having to install the database at all, you can run both it and the serv
     # volumes:
     #   - "./pgdata:/var/lib/postgresql/data"
 
-Go into the directory where you saved this file and run :code:`docker-compose up`. You will see the logs of both the database and PostgREST, and be able to access the latter on port 3000.
+Go into the directory where you saved this file and run :code:`docker-compose up`.
+You will see the logs of both the database and PostgREST, and be able to access the latter on port 3000.
+
+Go into the directory where you saved this file and run :code:`docker-compose up`.
```
Comment (steve-chavez) [inline install.rst]: Redundant line, maybe this will be the last change and then I'll merge.

Comment (ASVBPREAUBV) [reply install.rst]: Did you get it?

**Comment by steve-chavez** [comment]
Thanks @ASVBPREAUBV, added the corrections myself.

**Pull request merged**